### PR TITLE
feat(core): resize and reorder split-view

### PR DESCRIPTION
**Pull request by CatsJuice**

### CODE REVIEW

**Comment by CatsJuice** [comment]
* **#5994** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5994?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#5971** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5971?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CatsJuice and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/modules/workbench/view/workbench-root.tsx [hunk 1]
```diff
@@ -29,18 +28,35 @@ export const WorkbenchRoot = () => {
...
-      {views.map((view, index) => (
-        <WorkbenchView key={view.id} view={view} index={index} />
-      ))}
-    </div>
+      views={views}
+      renderer={panelRenderer}
+      onMove={onMove}
+    />
   );
+  // return (
```
Comment (EYHN) [inline packages/frontend/core/src/modules/workbench/view/workbench-root.tsx]: remove

**Comment by pengx17** [comment]
there's some style issues on windows

**Comment by CatsJuice** [comment]
we planned to handle responsive header later

**Review by EYHN — APPROVED** [review]

**Review by pengx17 — APPROVED** [review]

**Pull request merged**